BrentOzarULTD/SQL-Server-First-Responder-Kit
# Power BI Report Pages Update For Phone Layout

Update the FirstResponderKit Example for Training (pbix) and
FirstResponderKit (pbt - template) Power BI file(s) by including
visualizations and controls in Phone layout view. This allows the
report pages to be used in a more tailored view from a Phone with
the Power BI app.

Add Feature.

Update Pages:
 - Overview
 - Query Details
 - File Speeds
 - File Sizes
 - Perfmon
 - Wait Stats Anomalies
 - Perfmon Anomalies
 - Instructions

Fixes https://github.com/BrentOzarULTD/SQL-Server-First-Responder-Kit/issues/1809.

Changes proposed in this pull request:
 - Update the FirstResponderKit Example for Training (pbix) and FirstResponderKit (pbt - template) Power BI file(s) by including visualizations and controls in Phone layout view. 
 - This allows the report pages to be used in a more tailored view from a Phone with the Power BI app.

How to test this code:
 - Navigate to /SQL-Server-First-Responder-Kit/Power BI/
 - Open file [FirstResponderKit Example for Training.pbix]
 - Click View > Phone Layout
 - Repeat for all pages.
 - Scroll down to see content
 - Click File, Exit
 - Open file [FirstResponderKit.pbit]
 - Specify the Instance and Database, then click Load
 - Click View > Phone Layout
 - Click on each page Query Details > File Speeds > File Sizes > Perfmon > Wait Stats Anomalies > Perfmon Anomalies > Instructions
 -  Scroll down to see content
 - Click File > Publish > Publish to Power BI
 - Save your changes to your file system, specify a file name. ie. BlitzDev.pbix, then click Save.
 - Sign-in to the Power BI service from Power BI Desktop
 - Destination > click My workspace > Select
 - Replace dataset if prompted
 - Should say Success!, click Got it > File > Exit
 - On your mobile open the IOS App Store or Google Play Store
 - Search for Microsoft Power BI and press Open/Get/Install.
 - Sign-in to the Power BI service using the same credentials as in the previous step.
 - Press the Burger Menu > My workspace > Reports. 
 - Press the new report BlitzDev or whatever you decided to call it.
 - 💣 💥 
 - Press the Pages icon on the bottom right, switch to each page.
 - Press the Ellipses on the top right per visual > Expand to focus mode to see more details.
 - Press the < on the top left to return
 - Rotate your phone horizontally for the standard desktop view. 💣 
 - Rotate your phone vertically for the phone view. 💥 
 - 😄 ✔️ 

Has been tested on (remove any that don't apply): Yes.
 - Case-sensitive SQL Server instance
 - SQL Server 2017
